src/Magento/FunctionalTestingFramework/Util/TestGenerator.php

```diff
@@ -239,10 +255,17 @@ public function createAllTestFiles($testManifest = null, $testsToIgnore = null)
             $testsToIgnore = SuiteObjectHandler::getInstance()->getAllTestReferences();
         }
 
-        $testPhpArray = $this->assembleAllTestPhp($testManifest, $testsToIgnore);
-        foreach ($testPhpArray as $testPhpFile) {
+        $noError = $this->assembleAllTestPhp($testManifest, $testsToIgnore);
+        foreach ($this->cestPhpArray as $testPhpFile) {
             $this->createCestFile($testPhpFile[1], $testPhpFile[0]);
         }
+
+        if (!$noError) {
+            throw new TestFrameworkException("ERROR: " .
```

Comment:
Can this error message be unified across all error types:
- schema validation error
- annotation errors

Reply:
Schema errors will still break tests.
Annotation errors will not break tests as before.

Reply:
Does that mean:

- schema error results in no test generated? 
- annotation error cause the tests to still generate with erroneous annotations? 

Also from AC:
> Schema errors do not cause hard failure
> Annotation errors do not cause hard failure

Reply:
see reply in general comment